src/components/InputFields.resi

```diff
@@ -43,16 +43,16 @@ let selectInput: (
   ~fixedDropDownDirection: SelectBox.direction=?,
   ~customButton: React.element=?,
   ~buttonType: Button.buttonType=?,
-  ~dropdownCustomWidth: string=?,
+  ~dropdownCustomWidth: Js.String2.t=?,
```

Comment:
why `string`  type changed to `Js.String2.t`

Reply:
These files are generated by rescript extension.